shared/react-native/ios/Keybase/AppDelegate.m

```diff
@@ -43,6 +45,19 @@ - (BOOL)addSkipBackupAttributeToItemAtPath:(NSString *) filePathString
   return success;
 }
 
+- (void) createBkgReadableDir:(NSString*) path
+{
+  NSFileManager* fm = [NSFileManager defaultManager];
+  // Setting NSFileProtectionCompleteUnlessOpen makes the directory accessible as long as the user has
+  // opened it once unlocked. The files are still stored on the disk encrypted (note for the chat database, it
+  // means we are encrypting it twice), and are inaccessible otherwise.
+  NSDictionary* noProt = [NSDictionary dictionaryWithObject:NSFileProtectionCompleteUnlessOpen forKey:NSFileProtectionKey];
+  [fm createDirectoryAtPath:path withIntermediateDirectories:YES
+                 attributes:noProt
+                      error:nil];
+  [fm setAttributes:noProt ofItemAtPath:path error:nil];
```

Comment:
We create it at line 55 if it doesn't exist. This just picks it up if it was already there and sets the attribute (for people that already have the app installed).